python/graphstorm/model/utils.py

```diff
@@ -263,41 +302,71 @@ def load_model(model_path, gnn_model=None, embed_layer=None, decoder=None):
     if 'decoder' in checkpoint and decoder is not None:
         decoder.load_state_dict(checkpoint['decoder'])
 
-def load_sparse_embeds(model_path, embed_layer):
+def load_sparse_embeds(model_path, embed_layer, local_rank, world_size):
     """load sparse embeddings if any
 
+        Sparse embeddings are stored as:
+        $model_path/ntype0/sparse_emb_0.pt
+                           ...
+                           sparse_emb_N.pt
+        $model_path/ntype1/sparse_emb_0.pt
+                           ...
+                           sparse_emb_N.pt
+        ...
+
         Parameters
         ----------
         model_path: str
             The path of the model is saved.
         embed_layer: model
             A (distributed) model of embedding layers.
+        local_rank : int
+            Local rank
+        world_size : int
+            World size in a distributed env.
     """
     if embed_layer is None:
         return
     embed_layer = embed_layer.module \
         if isinstance(embed_layer, DistributedDataParallel) else embed_layer
 
     if len(embed_layer.sparse_embeds) > 0:
+        def load_sparse_emb(num_embs, ntype_path):
```

Comment:
it's worthwhile to have a unit test on this function. it's better to implement it as a top function in the script.